force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +135,21 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
+        Integer nbrOfVowels = 0;
+        Set<String> vowelsToConsider = new Set<String>{'a', 'e', 'i', 'o', 'u'};
+        
+        for (Integer i = 0; i < str.length(); i++) {
             // get individual characters from the string - hint use substring or split
+            Boolean isAVowel = vowelsToConsider.contains(str.substring(i, i + 1));
+            
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
+            if(isAVowel) {
```

Comment:
Instead of populating a Boolean, I would probably just increment the variable directly. Slightly more efficient here where that is all you are doing. 
` if (vowelsToConsider.contains(str.substring(i, i + 1))) { 
     nbrOfVowels++
   }`